crates/sui-kv-rpc/src/main.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use clap::Parser;
+use prometheus::Registry;
+use sui_kv_rpc::KvRpcServer;
+use sui_rpc_api::proto::rpc::v2beta::ledger_service_server::LedgerServiceServer;
+use sui_rpc_api::ServerVersion;
+use telemetry_subscribers::TelemetryConfig;
+use tonic::transport::Server;
+
+bin_version::bin_version!();
+
+#[derive(Parser)]
+struct App {
+    credentials: String,
+    instance_id: String,
+    #[clap(default_value = "[::1]:8000")]
+    address: String,
+    #[clap(default_value = "127.0.0.1")]
+    metrics_host: String,
+    #[clap(default_value_t = 8081)]
+    metrics_port: usize,
+}
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let _guard = TelemetryConfig::new().with_env().init();
+    let app = App::parse();
+    std::env::set_var("GOOGLE_APPLICATION_CREDENTIALS", app.credentials.clone());
+    let server_version = Some(ServerVersion::new("sui-kv-rpc", VERSION));
+    let registry_service = mysten_metrics::start_prometheus_server(
+        format!("{}:{}", app.metrics_host, app.metrics_port).parse()?,
+    );
+    let registry: Registry = registry_service.default_registry();
+    mysten_metrics::init_metrics(&registry);
```

Comment:
Maybe also nice to setup these grpc service metrics https://github.com/mystenlabs/sui/blob/main/crates/sui-rpc-api/src/metrics.rs here. It will require setting up a tower middleware like how its done on the fullnode https://github.com/mystenlabs/sui/blob/4c4d2f2ba486b1e26dd5dcebb363203ab5b51bb0/crates/sui-rpc-api/src/lib.rs#L216-L218